restaurants/views.py

```diff
@@ -12,7 +18,19 @@ class RestaurantListView(ListView):
     def get_queryset(self):
         queryset = super().get_queryset()
         self.filterset = RestaurantFilter(self.request.GET, queryset=queryset)
-        return self.filterset.qs
+        queryset = self.filterset.qs
+
+        sort_option = self.request.GET.get('sort_by')
+        if sort_option == 'cost_asc':
+            queryset = queryset.order_by('cost_for_two')
+        elif sort_option == 'cost_desc':
+            queryset = queryset.order_by('-cost_for_two')
+        elif sort_option == 'rating_asc':
+            queryset = queryset.order_by('rating')
+        elif sort_option == 'rating_desc':
+            queryset = queryset.order_by('-rating')
+
+        return queryset
 
```

Comment:
See if you can implement the django_filters for sorting too. I am attaching a screenshot of a filter used in testpress. Check the `o`  field Ignore everything else.

![Screenshot from 2024-08-08 12-06-48](https://github.com/user-attachments/assets/ab11fa0b-354e-4a50-a9a5-2f0fa03abd27)